rskj-core/src/main/java/co/rsk/peg/federation/PendingFederation.java

```diff
@@ -97,47 +98,42 @@ public PendingFederation addMember(FederationMember member) {
      * @return a Federation
      */
     public Federation buildFederation(
-        Instant creationTime,
-        long blockNumber,
-        BridgeConstants bridgeConstants,
-        ActivationConfig.ForBlock activations
+            Instant creationTime,
+            long blockNumber,
+            BridgeConstants bridgeConstants,
+            ActivationConfig.ForBlock activations
```

Comment:
Check your intellij config, looks like it's adding extra spaces

Reply:
it was me manually lol but ill rollback it